inference-engine/src/vpu/graph_transformer/src/stages/nms.cpp

```diff
@@ -10,6 +10,8 @@ namespace vpu {
 
 namespace {
 
+constexpr int boxesThreshold = 3650;
```

Comment:
Please add comment about this magic number both in NMS and StaticShapeNMS, because it is unclear.

Reply:
Thanks! Done.

Reply:
Still unclear why `3650` exactly

Reply:
Make it local to `getSHAVEsRequirementsImpl`